File: 7-kyu/transpose-two-strings-in-an-array/javascript/solution.js
const transposeTwoStrings = (array) => {
  let [left, right] = array;
  while (left.length < right.length) {
    left += " ";
  }
  while (right.length < left.length) {
    right += " ";
  }
  return [...left].map((elem, idx) => `${elem} ${right[idx]}`).join("\n");
};

File: 7-kyu/transpose-two-strings-in-an-array/python/solution.py
def transpose_two_strings(arr):
    left, right = arr
    while len(left) < len(right):
        left += ' '
    while len(right) < len(left):
        right += ' '
    return '\n'.join(f'{lhs} {rhs}' for lhs, rhs in zip(left, right))
